helper/strutil/strutil.go

```diff
@@ -174,6 +174,16 @@ func ParseArbitraryStringSlice(input string, sep string) []string {
 	return ret
 }
 
+// TrimStrings takes a slice of strings and returns a slice of strings
+// with trimmed spaces
+func TrimStrings(items []string) []string {
```

Comment:
You are right but I think it is safer to just copy into a new slice with one allocation.  Looking a little into the sorting code, it looks like this might be an optimization for the recursive methods.  I'm not sure it matters too much in this case.